### DISCUSSION TITLE
Why everyone who login var LDAP can view "Admin" tab?

### DISCUSSION THREAD

**Discussion by piapiamilapia** [eclipse-sw360/sw360#2621] [Q&A]
Why everyone who login var LDAP can view "Admin" tab?
I have configured LDAP in Instance Settings. Every user login using LDAP account. But all user can view "Admin" tab and execute  all menu items. I don't want the user who are "User"  of Primary Department Role can view the "Admin" tab.
How can I implement? 
Thanks.

**Comment by KoukiHama** [comment]
What is the organisation of users logged in via LDAP?
Where did these organisations set up?　In Liferay User setting?

**Reply by KoukiHama** [reply]
I can't verify this because I don't have an LDAP environment ready right now, but usually the Admin tab is hidden regardless of the login method.
